Modellen repository git - expliciete paden om werkbaar te maken voor meerdere gebruikers

jkaptein opened the pull request.

bastiaan-roos committed:
initial setup with tests

bastiaan-roos committed:
move tests

bastiaan-roos committed:
Merge branch 'main' into modellen-repository-git

bastiaan-roos committed:
some changes for development (on mac without threedigrid_builder)

bastiaan-roos committed:
improvements based on first test

- installation scripts
- forward and backward
- best order of events when working with github desktop

bastiaan-roos committed:
remove requirement to use git_model_repo in pythonpath

bastiaan-roos committed:
Merge branch 'main' into modellen-repository-git

bastiaan-roos committed:
fix for initialize model repo on network drive and move .cmd to .bat

bastiaan-roos committed:
move .env to root and add extra files to gitignore

bastiaan-roos committed:
select correct paths and run with windows

bastiaan-roos committed:
fix

bastiaan-roos committed:
fix install hooks under windows

jkaptein committed:
Add .gitattributes file for LFS

bastiaan-roos committed:
fix

bastiaan-roos committed:
Merge branch 'modellen-repository-git' of https://github.com/threedi/hhnk-threedi-tools into modellen-repository-git

bastiaan-roos committed:
minor changes and some docs

bastiaan-roos committed:
Merge branch 'modellen-repository-git' of https://github.com/threedi/hhnk-threedi-tools into modellen-repository-git

bastiaan-roos committed:
aanroep fix

bastiaan-roos committed:
only dump changed files and performance logging

bastiaan-roos committed:
set paths so it works with multiple installations in the python_path

bastiaan-roos committed:
path support for windows

bastiaan-roos committed:
correct path for change detection

jkaptein committed:
test

test

bastiaan-roos committed:
testplan (wip)

bastiaan-roos committed:
Merge branch 'main' into modellen-repository-git

bastiaan-roos committed:
add to gitignore

bastiaan-roos committed:
add extra dependency to pixi.toml

bastiaan-roos committed:
ruff fixes

bastiaan-roos committed:
ruff formatting

bastiaan-roos committed:
fix dependency, add mac

jkaptein committed:
update docs van azure

bastiaan-roos committed:
pixi config - fix macos (disable threedigrid_builder)

bastiaan-roos committed:
Merge branch 'modellen-repository-git' of https://github.com/threedi/hhnk-threedi-tools into modellen-repository-git

bastiaan-roos committed:
remove unused files

bastiaan-roos committed:
fix test

bastiaan-roos committed:
remove lfs file

bastiaan-roos committed:
possible fix tests?

bastiaan-roos committed:
possible fix tests?

bastiaan-roos committed:
add missing file for tests

bastiaan-roos committed:
small improvements

bastiaan-roos committed:
merge main

bastiaan-roos committed:
update pixi.toml

bastiaan-roos committed:
ruff formatting

bastiaan-roos committed:
ruff sort imports

bastiaan-roos committed:
fix test

bastiaan-roos committed:
refactor to Pathlib, docstring numpy style and typing on arguments

bastiaan-roos committed:
refactor test files to pathlib and few other review remarks

bastiaan-roos committed:
ruff styling

bastiaan-roos committed:
Merge branch 'main' into modellen-repository-git

bastiaan-roos committed:
update pixi.lock

bastiaan-roos committed:
review remarks resolved

- log => logger
- manual: location of models
- typo in test

bastiaan-roos committed:
ruff reformat

bastiaan-roos committed:
update bin files for use with pixi

bastiaan-roos committed:
fix test

bastiaan-roos committed:
fix test

bastiaan-roos committed:
fix typo and test

bastiaan-roos committed:
Modellen repo with pyogrio (#286)

* rewrite to pyogrio

* ruff import formatting

* fix tests with pyogrio

* fix tests for python39

* ruff fix

* add new python version to github testing

* type fix

jkaptein committed:
Merge branch 'main' into modellen-repository-git

jkaptein committed:
Update pixi.lock

jkaptein committed:
update git to pixi

jkaptein committed:
update 39py lock

jkaptein committed:
update_toml

jkaptein committed:
Update pixi.toml

jkaptein committed:
nu echt laatste comit final final v2

jkaptein committed:
pixi lock update v2 final

jkaptein committed:
update fix versiebeheer

versiebeheer had nog wat bugs
- python.bat had verkeerde term om naar pixi te kijken
- commitmessage keek niet naar goede folder om lengte te bepalen
- prepare commit msg gaf exit(1) error terwijl dit nog ingvuld moet worden

jkaptein committed:
update python bat to adjust for the location of the pixi env

jkaptein committed:
set pixi path to shared E model db drive

stuart-minion committed:
jelle updates

jkaptein committed:
update run_hhok.bat

jkaptein committed:
Merge branch 'main' into modellen-repository-git

stuart-minion reviewed (changes requested):
Ik heb nog steeds permissie errors.

De hardcoded paden blijven ook een probleem. Misschien moet de aanpak herzien?

jkaptein committed:
update commit_msg

- tekens lezen ipv openen bestand en lezen
- gitignore aanvullen

jkaptein committed:
Merge branch 'main' into modellen-repository-git

jkaptein committed:
Update gitignore ea

- setup new list of gitignores
- simplify python.bat removing errorlevels
- change pre_commit to show changed files
- pixi update

jkaptein committed:
Update install_hooks.py
